misc/config_tools/scenario_config/jsonschema/converter.py

```diff
@@ -389,7 +389,11 @@ def main():
         config_tools = config_tools.parent
 
     schema_file = config_tools / 'schema' / 'sliced.xsd'
-    json_schema_file = config_tools / 'configurator' / 'packages' / 'configurator' / 'build' / 'assets' / 'scenario.json'
+
+    json_schema_path = config_tools / 'configurator' / 'packages' / 'configurator' / 'build' / 'assets'
+    if not os.path.exists(json_schema_path):
+        os.makedirs(json_schema_path)
+    json_schema_file = json_schema_path / 'scenario.json'
```

Comment:
You can replace the above four lines with a single line below:

`json_schema.parent.mkdir(parents=True, exists_ok=True)`

Reply:
Updated.